torch/onnx/symbolic_opset9.py

```diff
@@ -1512,13 +1512,16 @@ def full(g, sizes, value, dtype, layout, device, pin_memory=False):
                     value_t=torch.tensor([const_value], dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
 
 
-@parse_args('v', 'f', 'i', 'v', 'v', 'v', 'v')
 def full_like(g, input, fill_value, dtype=None, layout=None, device=None, pin_memory=False, memory_format=None):
-    shape = g.op("Shape", input)
-    if dtype is None:
-        dtype = 6  # float
-    return g.op("ConstantOfShape", shape,
-                value_t=torch.tensor([fill_value], dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
+    if sym_help._is_value(fill_value):
+        dtype = 6 if dtype is None else dtype
+        tmp = zeros_like(g, input, dtype, layout, device)
+        return add(g, tmp, fill_value, g.op("Constant", value_t=torch.tensor(1)))
```

Comment:
Why is the constant 1 alpha arg neede?

Reply:
if alpha == 1 add function will return onnx::Add op:
https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset9.py#L85

Reply:
What if alpha is None (not passed)? I don't think it's necessary to pass 1.

Reply:
alpha cannot be passed to full_like. I agree it's not necessary to pass 1 to add op, but it's consistent with how add was used in other symbolic functions in the file.

Reply:
Actually this is not the case. Add() has been used without alpha in many instances. Still, this is not necessary.